Sources/Redis.swift

```diff
@@ -1559,30 +1559,30 @@ public class Redis {
     //  MARK: Helper functions
     //
     
-    private func redisBoolResponseHandler(_ response: RedisResponse, callback: (Bool, error: NSError?) -> Void) {
+    private func redisBoolResponseHandler(_ response: RedisResponse, callback: (Bool, _: NSError?) -> Void) {
         switch(response) {
         case .IntegerValue(let num):
             if  num == 0  || num == 1 {
-                callback(num == 1, error: nil)
+                callback(num == 1, _: nil)
```

Comment:
If the parameter has no name, you can just do `callback(num == 1, nil)` I think.